### DISCUSSION TITLE
After deploy in production, TailwindCSS and the Reusing Style not working.

### DISCUSSION THREAD

**Discussion by nicooh** [tailwindlabs/tailwindcss#11037] [Help]
After deploying my nextjs app with Tailwind, the stylesheet isn't loaded because the @apply directive is not translated with the respective properties and because the set of tailwind libraries in the style is not loaded. Everything works properly locally in development environnement. I can show the difference between production and development :
![image](https://user-images.githubusercontent.com/40459534/233113502-fe3f23bc-d62d-4345-a2f5-16c326af17df.png)
![image](https://user-images.githubusercontent.com/40459534/233113723-0fd21dac-95c9-4043-807d-b588cc196e5b.png)

**Comment by nicooh** [comment]
**tailwind.config.js**

```
module.exports = {
  important: true,
  content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
    "./sections/**/*.{js,ts,jsx,tsx}",

    "./App/**/*.{js,ts,jsx,tsx}",
    "./Pages/**/*.{js,ts,jsx,tsx}",
    "./Components/**/*.{js,ts,jsx,tsx}",
    "./Sections/**/*.{js,ts,jsx,tsx}",

    // Or if using `src` directory:
    "./src/**/*.{js,ts,jsx,tsx}",
  ]
```

**Reply by prantikseal** [reply]
Thanks this works
